From: Tony Jones <tonyj@suse.de>
Subject: change doc install path
References: none
Upstream: never

SuSE %{_docdir} is /usr/share/doc/packages. Change install locations to match.

---
 Makefile.am                                |    2 +-
 doc/Makefile.am                            |    2 +-
 doc/SystemTap_Tapset_Reference/Makefile.am |    4 ++--
 doc/beginners/Makefile.am                  |    2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -266,7 +266,7 @@ endif
 		| while read f; do if test -x $$f; then \
 			i_cmd="$(INSTALL_PROGRAM)"; else \
 			i_cmd="$(INSTALL_DATA)"; fi; \
-			$$i_cmd -D $$f $(DESTDIR)$(pkgdatadir)/examples/$$f; done)
+			$$i_cmd -D $$f $(DESTDIR)$(datadir)/doc/packages/systemtap/examples/$$f; done)
 	test -e $(DESTDIR)$(sysconfdir)/systemtap || mkdir -p $(DESTDIR)$(sysconfdir)/systemtap
 if BUILD_SERVER
 	test -e $(DESTDIR)$(localstatedir)/run/stap-server || mkdir -p $(DESTDIR)$(localstatedir)/run/stap-server
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -3,7 +3,7 @@
 
 AUTOMAKE_OPTIONS = subdir-objects
 PDF_FILES = tutorial.pdf langref.pdf
-DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/systemtap
+DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/packages/systemtap
 SUBDIRS = SystemTap_Tapset_Reference beginners
 
 if BUILD_DOCS
--- a/doc/SystemTap_Tapset_Reference/Makefile.am
+++ b/doc/SystemTap_Tapset_Reference/Makefile.am
@@ -1,9 +1,9 @@
 ## process this file with automake to produce Makefile.in
 

-DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/systemtap
+DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/packages/systemtap
 MAN_INSTALL_DIR = $(DESTDIR)$(mandir)/man3
-HTML_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/systemtap/tapsets
+HTML_INSTALL_DIR = $(DOC_INSTALL_DIR)/tapsets
 
 if XMLTO_STRINGPARAM
 XMLTOHTMLPARAMS=--stringparam use.id.as.filename=1 -m $(abs_srcdir)/generate-toc.xml
--- a/doc/beginners/Makefile.am
+++ b/doc/beginners/Makefile.am
@@ -2,7 +2,7 @@
 ## process this file with automake to produce Makefile.in
 
 
-DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/systemtap
+DOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/packages/systemtap
 SBG = SystemTap_Beginners_Guide
 BEGIN_INSTALL_DIR = $(DOC_INSTALL_DIR)/$(SBG)
 
